src/definitions/modules/dropdown.js

```diff
@@ -748,11 +759,13 @@ $.fn.dropdown = function(parameters) {
                 if(!Array.isArray(preSelected)) {
                     preSelected = preSelected && preSelected!=="" ? preSelected.split(settings.delimiter) : [];
                 }
-                $.each(preSelected,function(index,value){
-                  $item.filter('[data-value="'+value+'"]')
-                      .addClass(className.filtered)
-                  ;
-                });
+                if (module.is.multiple()) {
```

Comment:
I think, this condition is wrong, because also a single selection dropdown could have a prefilter via minCharacters. Adding this condition won't filter single selection dropdowns anymore

Reply:
Humm, this part of code is to hide the items from the menu who are already add in the `<input type="hidden">` : this case is only applicable when the dropdown is in multiple selection.

If you remove this condition, when dropdown is single selection and have already a selection (eg. `<input type="hidden" value="african-elephant">`), if the api is recall (to retrieve the items menu), the item previously selected in the menu is filtered and not visible : this is not what we want on single selection.

Maybe you think a similar option : `filterRemoteData`, but the filtering is not do here.